documentation/source/class.rst

```diff
@@ -29,6 +29,14 @@ additionally the ``members``, ``protected-members``, ``private-members`` and
 ``undoc-members``
    If specified, the undocumented members of the class will be displayed.
 
+``membergroups``
+  If specified, only the groups in a comma-delimired list following this
+  directive will be displayed.
+  
+``members-only``
+  This will allow to show only the members. It will remove group names,
+  descriptions as well as the inheritance information.
+
```

Comment:
If possible, could you also include two short example sections for both new options at the bottom of this page? You might be able to re-use some of the existing example code, possibly `examples/specific/group.h` and `examples/specific/{inheritence,cpp_inherited_members}.h`

In case new one(s) are better here is how to add one iirc:

* Add a new `$name.h` and `$name.cfg` in `examples/specific`, not that `.h` is used for C++ also here.
* Add the new `$name` to `projects` in `examples/specific/Makefile`
* Add the new `$name` to `breathe_projects` in `documentation/source/conf.py`.

For both examples a tiny one that renders once without the option and once with the option would be nice, so it's easy to see the difference.

As for the `doxygenstruct` page, you can add the same sections there with a simple reference to the sections on this page instead, as the example is effectively the same. A simple ```See the :ref:`doxygenclass documentation <class-example-membergroups>`.``` is perfectly fine.